src/bin/scheduler/setup/rcc.rs

```diff
@@ -260,20 +262,20 @@ fn enable_long_path_support(
     )
 }
 
-fn holotree_init(
+fn holotree_disable_sharing(
     global_config: &GlobalConfig,
     plans: Vec<Plan>,
 ) -> Result<(Vec<Plan>, HashMap<String, String>), Cancelled> {
     let mut command_spec =
         RCCEnvironment::bundled_command_spec(&global_config.rcc_config.binary_path);
-    command_spec.add_arguments(["holotree", "init"]);
+    command_spec.add_arguments(["holotree", "init", "--revoke"]);
     let mut succesful_plans = vec![];
     let mut failed_plans: HashMap<String, String> = HashMap::new();
 
     for (session, plans) in plans_by_sessions(plans) {
         debug!("Running {} for `{}`", command_spec, &session);
         let session_id = &format!(
-            "holotree_initialization_{}",
+            "holotree_disabling_{}",
```

Comment:
let's also call it `"holotree_disabling_sharing_{}"`

Reply:
Done